requirements.txt

```
@@ -0,0 +1,7 @@
+astropy
+pyyaml >= 5.1
+sqlalchemy
+git+https://github.com/lsst/dax_apdb@main#egg=lsst-dax-apdb
+git+https://github.com/lsst/utils@main#egg=lsst-utils
+git+https://github.com/lsst/resources@main#egg=lsst-resources
+git+https://github.com/lsst/felis@main#egg=lsst-felis
```

Can you modernize this syntax please?

```suggestion
lsst-felis @ git+https://github.com/lsst/felis@main
```
and the other lines. I see you are calling it `lsst-felis` and that does agree with the `pyproject.toml` in `felis` so that's a bit odd.

---

I tried modern syntax and got this error, switched it back to old one, still the same error. Will switch back to modern.